quasar/src/utils/open-url.js

```diff
@@ -9,7 +9,7 @@ export default (url, reject) => {
     }
     else if (navigator !== void 0 && navigator.app !== void 0) {
       return navigator.app.loadUrl(url, {
-        openExternal: true
+        openExternal: openExternal
```

Comment:
You are changing existing behaviour which breaks our API contract.

Reply:
Also, you will probably be passing "undefined" here if it is not given a default value.